src/scenes/gameuiscreen.ts

```diff
@@ -46,22 +50,37 @@ export class GameUiScreen extends Phaser.Scene {
         assert(this.drawnTilesContainer);
         this.drawnTilesContainer.removeAll(true);
         const myTiles: Nullable<Tile[]> = TileManager.getInHand();
-        assert(myTiles && this.uiBackground);
+        const session: Nullable<Session> = sessionManager.get();
+        const me: Nullable<Player> = PlayerManager.getMe();
+        const players: Nullable<Player[]> = PlayerManager.getAll();
+        assert(myTiles && this.uiBackground && session && players && me);
+        const playerCount: int = players.length;
 
         const nrTiles: int = myTiles.length;
         const tileSpacing: int =
             (this.uiBackground.width - nrTiles * 128) / (nrTiles + 1);
 
         for (let i: int = 0; i < nrTiles; i++) {
-            this.drawnTilesContainer.add(
-                this.add.image(
-                    this.uiBackground.getTopLeft().x +
-                        (tileSpacing + 128 / 2) +
-                        i * (128 + tileSpacing),
-                    ScreenHeight - 100,
-                    `tile${myTiles[i].type}`,
-                ),
+            const image: Phaser.GameObjects.Image = this.add.image(
+                this.uiBackground.getTopLeft().x +
+                    (tileSpacing + 128 / 2) +
+                    i * (128 + tileSpacing),
+                ScreenHeight - 100,
+                `tile${myTiles[i].type}`,
             );
+            if (session.turnIndex % playerCount === me.orderIndex) {
+                if (session.turnIndex === 0 && me.orderIndex === 0) {
```

Comment:
What is this if statement for? Why only add tiles to player with me.orderIndex = 0?

Reply:
oh yeah you‘re right, for a previous version it was needed but now its useless. I‘ll remove it